src/components/Transactions/Transactions.tsx

```diff
@@ -67,12 +76,14 @@ const Transactions = () => {
     error,
     loading,
     isListExpanded
-  } = useSelector(state => state.transactions);
-  const { pauseOffersIDs } = useSelector(state => state.offers);
+  } = useAppSelector(state => state.transactions);
```

Comment:
for appSelector we were adding a dedicated function to achieve something like: 
`useAppSelector(selectTransactions);` 
See selectFinalizePayment, selectOrder, etc.

Reply:
To do it I need to rewrite `offersSlice.js` to TypeScript, I will do it

Reply:
added it for offers, using it with transactions produces ts errors because whole transactions types needs to be added and slice rewritten, better do it in other task